harness/determined/tensorboard/util.py

```diff
@@ -10,6 +10,16 @@
     "*.pb",
 ]
 
+profiler_file_extensions = [
```

Comment:
question: why not upload everything in `profile/`?

Reply:
We do upload everything. But we only mangle things we recognize.

Unfortunately, the profilers generate files with a variable number of "extensions". So currently I just remove known extensions, add `#<rank>` to what is left, then add the extensions back. An alternative would be to insert `#<rank>` before the first dot. However, if we ever get a host name with embedded dots we would be screwed.